SEED-platform/seed
# Fixes Timeline Bugs

#### Any background context you want to provide?
Small bugs for the timeline and scenario/measures sections

#### What's this PR do?
- audit dates are by default 'YYYY-MM-DD'. Change to 'YYYY/MM/DD' to match "upload date"
- adds expand/contract all to inventory detail scenarios and measures 
- CSS spacing

#### How should this be manually tested?
- check inventory detail scenarios and measures audit date formatting
- try expand all / contract all button on detail scenarios and measures. Make sure expanded measure ui-grids are not empty

#### What are the relevant tickets?
#3843 

#### Screenshots (if appropriate)
